models/yiiModels/YiiProjectModel.php

```diff
@@ -180,7 +177,10 @@ public function attributesToArray() {
         $elementForWebService[YiiProjectModel::DESCRIPTION] = $this->description;
         $elementForWebService[YiiProjectModel::OBJECTIVE] = $this->objective;
         $elementForWebService[YiiProjectModel::PARENT_PROJECT] = $this->parentProject;
-        $elementForWebService[YiiProjectModel::WEBSITE] = $this->website;
+        //SILEX:conception
+        // use a getter
+        //\SILEX:conception
+        $elementForWebService[YiiProjectModel::WEBSITE] = ($this->website === "") ? null : $this->website;
```

Comment:
Why this field in particular ? Don't you have the same problem with all ? 
Could you check please ?